app/src/main/java/com/example/lister/ListDatabaseHelper.java

```diff
@@ -102,4 +104,27 @@ public void addList(String list, int id) {
 
         return lists;
     }
+
+    public int updateListName(String list, int id){
+        SQLiteDatabase db = this.getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(ListerDatabase.List.LIST_NAME, list);
+        return db.update(ListerDatabase.List.TABLE_NAME, values,
+                ListerDatabase.List._ID + "= " + id, null);
+    }
```

Comment:
Note: stolen from Kevin's branch.
The code below was also stolen from him but changed to only delete the specified item instead of the entire table.